src/6_branch.js

```diff
@@ -1957,14 +1957,29 @@ Branch.prototype['setAPIResponseCallback'] = wrap(callback_params.NO_CALLBACK, f
  */
 Branch.prototype['setDMAParamsForEEA'] = wrap(callback_params.CALLBACK_ERR, function(done, eeaRegion, adPersonalizationConsent, adUserDataUsageConsent) {
 	try {
-		var dmaObj = {};
-		dmaObj.eeaRegion = eeaRegion || false;
-		dmaObj.adPersonalizationConsent = adPersonalizationConsent || false;
-		dmaObj.adUserDataUsageConsent = adUserDataUsageConsent || false;
+		const validateParam = (param, paramName) => {
+			if (!utils.isBoolean(param)) {
+				console.warn(`setDMAParamsForEEA::DMA parameter ${paramName} must be boolean`);
+				return false;
+			}
+			return true;
+		};
+		const isValid = (validateParam(eeaRegion, "eeaRegion") &&
+		validateParam(adPersonalizationConsent, "adPersonalizationConsent") &&
```

Comment:
Let's format this line using prettier, or something.

Reply:
and this is the formatter approved code?

Reply:
i mean.. i can't think that any formatter is ok putting continued expression as same line as variable declaration